migrations/sqls/20190726234757-programs-up.sql

```diff
@@ -0,0 +1,155 @@
+/* Replace with your SQL commands */
```

Comment:
This file is somewhat scary -- it's trying to reconcile the services that we have in the db already with the services that are in https://docs.google.com/document/d/1OgZf5acOTpbd-O0Up8ItRlW81Vspq8Rc6yrES1JlwLk/edit. Also it's creating a programs table and connecting all services to a program.